crates/lib/src/tags/include_tag.rs

```diff
@@ -82,7 +82,7 @@ impl ParseTag for IncludeTag {
     }
 }
 
-#[cfg(test)]
+#[cfg(test_deactivate)]
```

Comment:
This meant to be here?

Reply:
Uh what? Oo I don't have a clue on how this happen x)